library/Fission/Internal/Fixture/Key/RSA2048.hs

```diff
@@ -0,0 +1,18 @@
+{-# OPTIONS_GHC -fno-warn-incomplete-uni-patterns #-}
+
+module Fission.Internal.Fixture.Key.RSA2048
+  ( pk
+  , rawPK
+  ) where
+
+import           Servant.API
+
+import           Fission.Prelude
+import qualified Crypto.PubKey.RSA as RSA
+import           Fission.Internal.Orphanage.RSA2048.Public ()
+
+pk :: RSA.PublicKey
+Right pk = parseUrlPiece rawPK
+
+rawPK :: Text
+rawPK = "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAoZeA/ijID5iZqPCBJFdml7wfxY/bLKfEIaacpq7CW/Y4bm/v/peLGiRMD1+k9IrmoA0I9h8hczfegaHWCEPovlzxu5YErbtuaI14Yqm90Nq1T0wLNysdjLx995G7w4W+FGNSoIivxfmeY8RKt0v2Z8EykiWA81vSbberXOqyDeAo0cRqbvFVynS4oUUQZOeaKj+SWMbYtG1nAnu5frNknOCIQVqUr6NXEwXmSbk/vO/CqkqHRhv4p9wRWQbk7/GRMiaVu0m9qV4rlD1F9QL7Q9HYi16k7WmpSW7Wlw330ZgH5RLt4pTcuTyGMt1loCBFwPWSO5DbFDwS+EgldePMqwIDAQAB"
```

Comment:
We actually have a few of these already (as key pair fixtures). The file is horrifying, but works. Not sure if that helps or not for this use case, though

Reply:
>We actually have a few of these already (as key pair fixtures)

oh really? where?

Reply:
This file: https://github.com/fission-suite/fission/blob/b9022546ddf4074d35462d328dd72ec99e49473d/library/Fission/Internal/RSA2048/Pair/Types.hs#L12